service/eid/eid-facility-mappings.json

```diff
@@ -2501,6 +2501,132 @@
+    "mrsDisplay": "Alale health centre",
+    "description": "This is a module under KCRH",
+    "mflCode": "16367",
+    "county": "West Pokot",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "d0d133f1-22f3-49c3-b9f1-d2f64944eb0d": {
+    "mrsId": 425,
+    "mrsDisplay": "Amakuriat dispensary",
+    "description": "This is a module  under KCRH",
+    "mflCode": "14198",
+    "county": "West Pokot",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "0a2d9adf-d758-4c81-8429-2ff54e2c58d6": {
+    "mrsId": 427,
+    "mrsDisplay": "Lomut dispensary ",
+    "description": "This is a module  under KCRH",
+    "mflCode": "15073",
+    "county": "West Pokot",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "4949293e-7b5c-4359-8a55-1196a578143b": {
+    "mrsId": 466,
+    "mrsDisplay": "Top Station Dispensary ",
+    "description": "This is a module under KCRH",
+    "mflCode": "15734",
+    "county": "transzoia",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "6666bb3c-1500-4320-9c46-78efc5bbaee0": {
+    "mrsId": 418,
+    "mrsDisplay": "Rafiki Health center ",
+    "description": "This is a module under KCRH",
+    "mflCode": "22624",
+    "county": "transzoia",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "1dc987ac-5396-4498-9e87-a344aeb4b290": {
+    "mrsId": 280,
+    "mrsDisplay": "Kiminini Health center",
+    "description": "This is a module under KCRH",
+    "mflCode": "16362",
+    "county": "transzoia",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "29664340-6318-4808-8adb-6be98ddcb81b": {
+    "mrsId": 417,
+    "mrsDisplay": "Kobos Health center ",
+    "description": "This is a module under KCRH",
+    "mflCode": "14958",
+    "county": "transzoia",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "7a948722-45db-4b55-bbf0-0637644b0228": {
+    "mrsId": 431,
+    "mrsDisplay": "Kapsitwet Dispensary",
+    "description": "This is a module for under KCRH",
+    "mflCode": "14762",
+    "county": "transzoia",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "0db6a78c-6b64-420c-bc30-ac403417c237": {
+    "mrsId": 434,
+    "mrsDisplay": "Chesoi Health Centre",
+    "description": "This is a module under KCRH",
+    "mflCode": "14385",
+    "county": "EMC",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "ce8089a9-d105-4d79-8a0a-d4a389d691b4": {
+    "mrsId": 435,
+    "mrsDisplay": "Chesongoch Health Centre",
+    "description": "This is a module under KCRH",
+    "mflCode": "14386",
+    "county": "EMC",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "5794ab51-f531-46b8-acc0-51d77ff23760": {
+    "mrsId": 436,
+    "mrsDisplay": "Kaparon Health Centre",
+    "description": "This is a module under KCRH",
+    "mflCode": "16335",
+    "county": "EMC",
+    "cityVillage": "",
+    "eidDisplay": ""
+  },
+  "086e6f2e-b67e-4926-a8fa-23999a7b7af1": {
+    "mrsId": 433,
+    "mrsDisplay": "Kaptarakwa Sub County Hospital",
+    "description": "This is a module under KCRH",
```

Comment:
For description, we can re-use the mrsDisplay

Reply:
okay
on it

Reply:
Done